src/components/AdvanceAnalyticsV2/AnalyticsV2Page.jsx

```diff
@@ -17,7 +17,13 @@ import { useEnterpriseAnalyticsAggregatesData } from './data/hooks';
 import { GRANULARITY, CALCULATION } from './data/constants';
 import { formatTimestamp } from '../../utils';
 
-const PAGE_TITLE = 'Analytics';
+const messages = defineMessages({
+  pageTitle: {
+    id: 'analytics.v2.page.title',
+    defaultMessage: 'Analytics',
+    description: 'Title of the analytics v2 page',
```

Comment:
I think we need to remove all references to V2. We don't have separate V1 and V2 Analytics, we just have Analytics.